reference/docs-conceptual/dsc/quickstarts/website-quickstart.md

```diff
@@ -112,6 +112,10 @@ The `Start-DscConfiguration` cmdlet tells the [Local Configuration Manager (LCM)
 which is the engine of DSC, to apply the configuration.
 The LCM does the work of calling the DSC resources to apply the configuration.
 
+> [!NOTE] To allow DSC to run, Windows needs to be configured to receive PowerShell remote commands,
```

Comment:
Reflow wrapped this incorrectly. You have to watch out for NOTE blocks.

Reply:
I wondered why I had written it that way and then realized what had happened. Lesson learned